processor/transaction/package_tests/TestProcessTransactionFull.approved.json

```diff
@@ -110,6 +110,12 @@
                     "username": "foo"
                 }
             },
+            "labels": {
+                "organization_uuid": "9f0e9d64-c185-4d21-a6f4-4673ed561ec8"
+            },
+            "process": {
+                "args": "node server.js"
```

Comment:
`process.args` is meant to be an array of individual terms.

Hmmm actually the spec formats the example as an array, but doesn't state this clearly otherwise. We'll have to clarify that field description...

Anyhow, doing it as an array of terms enables the possibility of doing keyword searches on any of the args, which I think is desirable.

WDYT, @ruflin?

Reply:
I completely misunderstood that - I thought the docs were saying it's one big keyword and ended up [implementing](https://github.com/elastic/apm-server/pull/1609/files#diff-f7e31f1f55e1a508546a00f6ae75a26fR35) as such.  Thanks for spotting this @webmat.

Reply:
yes, should be an array on individual keywords.